### DISCUSSION TITLE
Introduce a one-liner comment syntax?

### DISCUSSION THREAD

**Discussion by mimoo** [ocaml/ocaml#12066] [Features]
Hey hey. I've been wondering why OCaml doesn't have a one liner comment syntax, something like `#` or `//` in other languages. I find it quite tedious to always have to close a comment opener for short comments, or quick commenting.

**Comment by dra27** [comment]
See #671 🍿

**Comment by Octachron** [comment]
Let me close this duplicate issue.

**Comment by mimoo** [comment]
@Octachron can you link to the other issue that's open?

**Comment by Octachron** [comment]
You can discuss on the closed issue and see if you can bring enough arguments to convince people to reopen the issue.

**Comment by Octachron** [comment]
I forgot that #671 was a PR . Let's avoid putting more pressure on the PR author and let's use this issue for discussion purpose.

This doesn't change the fact that there is no plan to introduce one-line comment without a well-though proposition that preserves backward compatibility.

**Comment by mimoo** [comment]
I'm fine with `(*)` personally, I don't care much as long as there's something

**Comment by damiendoligez** [comment]
@Octachron should this be turned into a discussion ?

**Reply by Octachron** [reply]
That sounds like a good idea, done.

**Comment by mcint** [comment]
For greater context on this 9 year old request (#671) and discussion (https://github.com/ocaml/ocaml/issues/11689#issuecomment-1298897183)

Per https://hyperpolyglot.org/ml
### grammar and invocation
. | sml | ocaml | f# | haskell
-- | -- | -- | -- | --
end-of-line comment | none | none | // comment | -- comment
multiple line comment | (* commentanother comment *) | (* commentanother comment *) | (* commentanother comment *) | {- commentanother comment -}

Outside of ml family, of the languages present on the site, selected implicitly for wide use or relevance, `// ...` is the most common "end-of-line" comment syntax.
